error_reporting/google/cloud/error_reporting/client.py

```diff
@@ -12,13 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Client for interacting with the Stackdriver Logging API"""
+"""Client for interacting with the Stackdriver Error Reporting API"""
 
+import os
 import traceback
 
-import google.cloud.logging.client
+try:
```

Comment:
It also is not entirely clear to me why this will raise ImportError. The `_gax_` module is present, so it has to raise `ImportError` somehow on its import. Which one of its imports is going to raise, and in what situation will it do so?